addons/api/.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
   env: {
     browser: true,
   },
+  globals: { __electronLog: 'true' },
```

Comment:
It's unfortunate we need this but it should only be used in the `client-daemon-handler`...but this is a very leaky implementation detail now. 

Open to other thoughts, I could always make a call to the ipc handler to make log statements in the main process instead of renderer as well.